src/components/MenuPanel/MenuPanel.js

```diff
@@ -260,6 +260,8 @@ const Wrap = styled(animated.div)`
   width: 90vw;
   height: 100vh;
   min-width: 300px;
+  flex-shrink: 0;
+  flex-grow: 0;
```

Comment:
@bpierre What do you think about the `flex` shorthands; do you think this would be clearer as `flex: none;`?

Reply:
I usually avoid `flex` for clarity, but I don’t really have a strong opinion, so let’s use it if @AquiGorka agrees too 👍

Reply:
I'm ok with either approach.

Reply:
Done!